.github/workflows/linux.yml

```diff
@@ -0,0 +1,50 @@
+name: Linux
+
+on:
+  push:
+    branches:
+      - master
+      - 'support/*'
+  pull_request: {}
+
+concurrency:
+  group: linux-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      max-parallel: 1
```

Comment:
Given that this replaces more than one workflow, I think we can use more than one here.

Reply:
@nilmerg Do you agree and how much more?

Reply:
If my calculation is correct, `2`, as per the previous definition.